test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -564,9 +563,24 @@ BOOST_AUTO_TEST_CASE(rndCode)
 {
 	try
 	{
-		cnote << "Testing Random Code: ";
-		std::string code = test::RandomCode::generate(1000);
-		cnote << code;
+		test::RandomCodeOptions options;
+		std::string code = test::RandomCode::generate(1000, options);
+		BOOST_REQUIRE(!code.empty());
+	}
+	catch(...)
+	{
+		BOOST_ERROR("Exception thrown when generating random code!");
+	}
```

Comment:
Can't there be a way to show what kind of exception it was.  At least show `e.what()`.